pkg/controller/bootstrap/tokencleaner_test.go

```diff
@@ -100,3 +100,40 @@ func TestCleanerNotExpired(t *testing.T) {
 
 	verifyActions(t, expected, cl.Actions())
 }
+
+func TestCleanerExpiredAt(t *testing.T) {
+	cleaner, cl, secrets, err := newTokenCleaner()
+	if err != nil {
+		t.Fatalf("error creating TokenCleaner: %v", err)
+	}
+
+	secret := newTokenSecret("tokenID", "tokenSecret")
+	addSecretExpiration(secret, timeString(3*time.Second))
```

Comment:
making this identical to the sleep below seems likely to introduce flakes. suggest an expiration of 2 seconds here, and a polling check below checking every 100ms (with a timeout of `wait.ForeverTestTimeout`) until the queue length is 1

Reply:
make sense, updated. thanks ：） ：)